IntelliTect.TestTools.TestFramework/IntelliTect.TestTools.TestFramework/TestBuilder.cs

```diff
@@ -208,12 +208,17 @@ public void ExecuteTestCase()
 
         private static string GetObjectDataAsJsonString(object obj)
         {
-
+            // JsonSerializer.Serialize has some different throw behavior between versions.
+            // One version threw an exception that occurred on a property, which happened to be a Selenium WebDriverException.
+            // In this one specific case, catch all exceptions and move on to provide standard behavior to all package consumers.
+            // TL;DR: we don't want logging failures to interrupt the test run.
             try
             {
                 return JsonSerializer.Serialize(obj, new JsonSerializerOptions { WriteIndented = true });
             }
-            catch (JsonException e)
+#pragma warning disable CA1031 // Do not catch general exception types
+            catch (Exception e)
+#pragma warning restore CA1031 // Do not catch general exception types
```

Comment:
This seems fine, however I would advocate not just logging `e.Message` this can be problematic if the useful information is contained inside of its inner exception. I would use `e.ToString()` instead

Reply:
Thank you! I always get those two confused. I'll make that change real quick.

Reply:
Okay, change made. One of the versions I tested did throw an exception with an inner exception, so this should be more helpful. Thanks again!